aws-android-sdk-apigateway-test/.idea/workspace.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
For comparison, we ignore this file in [`amplify-android`'s `.gitignore`](https://github.com/aws-amplify/amplify-android/blob/master/.gitignore#L41). Maybe we should in `aws-sdk-android`, too?